.github/workflows/test.yml

```diff
@@ -22,6 +22,8 @@ jobs:
           node-version-file: 'js/.nvmrc'
           cache: 'npm'
           cache-dependency-path: 'js/package-lock.json'
+      - name: Lint JS code. If fails, run `just fmt-js` to reformat, and `just lint-js` to see issues locally
+        run: just lint-js
```

Comment:
Can linting be moved to a separate job? It is best that we see whether tests are passing or failing and not have them hidden behind a linting error.